syncer/filter.go

```diff
@@ -29,26 +29,27 @@ import (
 // - any schema of table names is system schema.
 // - any table name doesn't pass block-allow list.
 // - type of SQL doesn't pass binlog filter.
-func (s *Syncer) skipQuery(tables []*filter.Table, stmt ast.StmtNode, sql string) (bool, error) {
+// skipQuery return (ignored, isBaList, error).
```

Comment:
isBaList means in block list or allow list?

Reply:
Whatever block list or allow list may lead to ignore this query event.